server/src/main/java/com/linecorp/centraldogma/server/internal/api/converter/WatchRequestConverter.java

```diff
@@ -60,11 +60,17 @@ public WatchRequest convertRequest(
             ServiceRequestContext ctx, AggregatedHttpRequest request, Class<?> expectedResultType,
             @Nullable ParameterizedType expectedParameterizedResultType) throws Exception {
 
-        final String ifNoneMatch = request.headers().get(HttpHeaderNames.IF_NONE_MATCH);
+        String ifNoneMatch = request.headers().get(HttpHeaderNames.IF_NONE_MATCH);
         if (isNullOrEmpty(ifNoneMatch)) {
             return null;
         }
 
+        if (ifNoneMatch.startsWith("\"") && ifNoneMatch.endsWith("\"")) {
```

Comment:
Could you add a comment that describes the valid three cases?

Reply:
I add javadoc like:
```
    /**
     * Converts the specified {@code request} to a {@link WatchRequest} when the request has
     * {@link HttpHeaderNames#IF_NONE_MATCH}. {@code null} otherwise.
     *
     * {@link HttpHeaderNames#IF_NONE_MATCH} must be integer, ETag and Weak Comparison ETag.
     */
```

Reply:
It's not an integer it also can be a String such as `head` or `init`.
Instead of adding it to Javadoc, how about adding just comments so that reviewers easily notice the valid cases?
```java
// Thee below cases are valid:
// - <revision> (for backward compatibility)
// - "<revision>"
// - W/"<revision>"
```